chore: remove workaround for sentry-dotnet issue #3141 (#85)
getsentry/sentry-powershell: CHANGELOG.md

## Unreleased

### Fixes

> - Remove workaround for Dotnet SDK issue causing a hang on process exit - automatic flush should work now. ([#85](https://github.com/getsentry/sentry-powershell/pull/85))

Reviewer: suggestion: Swap the two parts of the note so that the actual fix/improvement comes first.

e.g.
```suggestion
- Automatic flush by removing workaround for Dotnet SDK issue causing a hang on process exit. ([#85](https://github.com/getsentry/sentry-powershell/pull/85))
```
